web3-functions/redstone/index.ts

```diff
@@ -13,8 +13,24 @@ const ORACLE_ABI = [
   "function getLivePricesAndTimestamp(bytes32[]) public view returns (uint256[], uint256)",
 ];
 
+const REDSTONE_PRIMARY_PROD = "redstone-primary-prod";
+const REDSTONE_MAIN_DEMO = "redstone-primary-prod";
+// https://github.com/redstone-finance/redstone-oracles-monorepo/blob/main/packages/sdk/src/data-services-urls.ts
+const DEV_GWS = [
+  "https://oracle-gateway-1.b.redstone.finance",
+  "https://d33trozg86ya9x.cloudfront.net", // https://github.com/gelatodigital/w3f-redstone-poc-v2/blob/main/web3-functions/redstone/index.ts#L32
+];
+const PROD_GWS = [
+  "https://oracle-gateway-1.a.redstone.finance",
+  "https://oracle-gateway-2.a.redstone.finance",
+];
+const REDSTONE_DATA_SERVICES_URLS: Record<string, string[]> = {
+  REDSTONE_PRIMARY_PROD: PROD_GWS,
+  REDSTONE_MAIN_DEMO: DEV_GWS,
```

Comment:
Preferable:
```suggestion
  [REDSTONE_PRIMARY_PROD]: PROD_GWS,
  [REDSTONE_MAIN_DEMO]: DEV_GWS,
```
or, modify L86 and L97 to be one of:
```
// L86:
REDSTONE_DATA_SERVICES_URLS.REDSTONE_PRIMARY_PROD // or
REDSTONE_DATA_SERVICES_URLS["REDSTONE_PRIMARY_PROD"]
...
// L97:
REDSTONE_DATA_SERVICES_URLS.REDSTONE_MAIN_DEMO // or
REDSTONE_DATA_SERVICES_URLS["REDSTONE_MAIN_DEMO"]
```